### PR TITLE
styled welcome screen with select language modal

### PR DESCRIPTION
**Pull request by stephaniewong2**
### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy!"
IMG_9718
IMG_9719

## How to review
[//]: # "Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?"

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### Online sources
[//]: # "Optional - copy links to any tutorials or documentation that was useful to you when working on this PR"

### Related PRs
[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @surajr10 @themiachou8

### CODE REVIEW

File: src/firebase/auth.ts [hunk 1]
```diff
@@ -141,4 +144,13 @@ export const signUserOut = async (dispatch: AuthDispatch) => {
...
-};
\ No newline at end of file
+};
+
+export const updateLanguage = async (
+  dispatch: AuthDispatch,
+  langUpdate: React.Dispatch<React.SetStateAction<Dictionary>>,
+  params: { language: string },
+) => {
+  langUpdate(checkAndGetLang(params.language));
```
Comment (surajr10) [inline src/firebase/auth.ts:163]: when calling this function make sure to also run a firebase query to change the language preference for the user document.

**Review by surajr10 — APPROVED** [review]
